### PR TITLE
Added Pull request template and gitignore

### PR DESCRIPTION
**Pull request by f7deleon**
Added  Pull request template and .gitignore

### Trello card :
https://trello.com/c/IEAXOiJM

### CODE REVIEW

**Review by mirland — CHANGES_REQUESTED** [review]
Add a better pull request name, a better description and assign the reviewers

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Short summary of your changes. Explain what have you done and comment the decisions that you may have taken.
```
Comment (mirland) [inline docs/PULL_REQUEST_TEMPLATE.md]: change the file folder too

**Review by mirland — CHANGES_REQUESTED** [review]

**Review by FerrariBruno — CHANGES_REQUESTED** [review]
Maybe you can add a basic `README.md` file, there are a ton of examples in the [Xmartlabs repos](https://github.com/xmartlabs). This is just for you to experience Markdown, it doesn't need to be a super README.

**Review by FerrariBruno — APPROVED** [review]

**Review by mirland — APPROVED** [review]
The name of the pull request is still wrong, rename it, you have to tell what are you doing

**Pull request merged**